inputs/LiMetal_PorousSep_spmLiAir.yaml

```diff
@@ -47,20 +47,21 @@ cell-description:
           D_k: 2e-13 # can't find (assume low?)
   cathode:
     class: 'metal_air_single_particle'
```

Comment:
You probably need to change this to `metal_air_discret` in order for it to run, correct?